internal/extension/extension.go

```diff
@@ -177,28 +178,17 @@ func (em *ExtensionManager) SendEndInvocationRequest(ctx context.Context, functi
 		}
 		if samplingPriority, ok := ctx.Value(DdSamplingPriority).(string); ok {
 			req.Header.Set(string(DdSamplingPriority), samplingPriority)
-		} else {
-			logger.Error(fmt.Errorf("could not get sampling priority from DdSamplingPriority"))
 		}
 	} else {
 		spanContext := functionExecutionSpan.Context()
-		req.Header.Set(string(DdTraceId), fmt.Sprint(spanContext.TraceID()))
+		req.Header.Set(string(DdTraceId), fmt.Sprint(spanContext.TraceIDLower()))
 		req.Header.Set(string(DdSpanId), fmt.Sprint(spanContext.SpanID()))
 
 		// Try to get sampling priority
-		// Check if the context implements SamplingPriority method
-		if pc, ok := spanContext.(interface{ SamplingPriority() (int, bool) }); ok && pc != nil {
-			if priority, ok := pc.SamplingPriority(); ok {
-				req.Header.Set(string(DdSamplingPriority), fmt.Sprint(priority))
-			} else {
-				logger.Error(fmt.Errorf("could not get sampling priority from spanContext.SamplingPriority()"))
-			}
+		if priority, ok := spanContext.SamplingPriority(); ok {
+			req.Header.Set(string(DdSamplingPriority), fmt.Sprint(priority))
 		} else {
-			if priority, ok := getSamplingPriority(functionExecutionSpan); ok {
-				req.Header.Set(string(DdSamplingPriority), fmt.Sprint(priority))
-			} else {
-				logger.Error(fmt.Errorf("could not get sampling priority from getSamplingPriority()"))
-			}
+			logger.Error(fmt.Errorf("could not get sampling priority from spanContext.SamplingPriority()"))
```

Comment:
`spanContext.SamplingPriority` is directly available in dd-trace-go v2. No longer need the handling for the `adapter` case